src/modules/register/Register.tsx

```diff
@@ -4,12 +4,21 @@ import { useTechnologiesProvider } from 'core/technologies';
 
 import RegisterForm from './register-form';
 
+import Navbar from "shared/components/navbar";
+import Footer from "shared/components/footer";
```

Comment:
You can import like { Navbar, Footer } from shared/components. Just add exports in shared/components/index.ts file - remember about alphabetic order